build.gradle

```diff
@@ -25,4 +25,5 @@ test {
     testLogging {
         events "passed", "skipped", "failed"
     }
+    systemProperties = System.getProperties()
```

Comment:
why do you need that?

Reply:
I need the systemProperties = System.getProperties() to send the browser in the terminal